plasmapy/particles/particle_class.py

```diff
@@ -111,13 +145,44 @@ def mass(self) -> Union[u.Quantity, Real]:
     def charge(self) -> Union[u.Quantity, Real]:
         raise NotImplementedError
 
+    @property
+    def json_dict(self) -> dict:
+        """
+        Provides the standard format for JSON representation of particles which
+        inherit from AbstractParticle.
+
+        Keys:
+            plasmapy_particle - Specifies that this JSON is for some particle
+                type - The type of the particle (e.g Particle or CustomParticle)
+                module - Specifies the module where the class was declared
+                date_created - Presents the UTC date and time of serialization
+                __init__ - contains a dictionary of the arguments used to create the particle
+                    args - non keyword arguments used to create a particle
+                    kwargs - keyword arguments used to create a particle
```

Comment:
I don't think this will render well in the RTD documentation, but I also just realized Sphinx isn't even rendering this for RTD. 😳 I'll look into this and will probably need push through a new PR for it.

Reply:
I see. If possible, could I get a sample format for how I could structure the keys of the JSON?

Reply:
I checked out the details at [Sphinx RTD Docstring Format](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format) 

Is this suitable for use here?

Reply:
> I checked out the details at [Sphinx RTD Docstring Format](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format)
> 
> Is this suitable for use here?

Probably better for use with methods and class docstrings as opposed to describing the dictionary though